### ISSUE TITLE
Requirements icons do not read in screen reader need alt text

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/scholarsphere-3#640]
The check and x icons should have alt text to indicate to screen readers that they still have stuff to do.

**Comment by carolyncole** [comment]
We need to change the java script to show hide actual icons and not after items in css.  The icons need alt text or real text so they are read.

**Comment by mtribone** [comment]
@awead My thought was to hook onto the checks here: https://github.com/projecthydra/sufia/blob/7.2-migration/app/assets/javascripts/sufia/save_work/save_work_control.es6#L107

and inserting some text like:
https://github.com/psu-stewardship/scholarsphere/blob/8b603196665da8e7a0db769b8a1169d80b06f54a/app/assets/javascripts/scholarsphere/viz_widget.js
